vignettes/web_only/BCRpipeline.Rmd

```diff
@@ -44,9 +44,15 @@ The pipeline involves five steps:
 
     This step involves preparation for phylogenetic and somatic hypermutation analysis.
 
+
 5. **Phylogenetic analysis.**
 
-    This final step provides phylogeny reconstruction and trunk length calculation (by running the PHYLIP package).
+    This step provides phylogeny reconstruction and trunk length calculation (by running the PHYLIP package).
+    
+
+6. **Somatic hypermutation analysis.**
+
+    At this step, clonotype and germline sequences are compared to count the number of mutations.
```

Comment:
At this step, we compare clonotype and germline sequences to detect and count the number of mutations in smth.